docs/developer-docs/gas-cost.md

```diff
@@ -38,26 +38,28 @@ This documentation will use the following units to measure and calculate the num
 
 ## The ICP reverse gas model
 
-In contrast to other blockchains where users pay for their transactions, ICP charges the smart contract canister for the resources it consumes. This allows developers to provide a smoother user experience, as users are free from tedious tasks such as signing and approving every transaction they perform within the application. This cost model is known as ICP's 'reverse gas model'.
+ICP charges the smart contract canister for the resources it consumes. This allows developers to provide a smoother user experience, as users are free from tedious tasks such as signing and approving every transaction they perform. This cost model is known as ICP's 'reverse gas model'.
 
 :::info
-While this may be referred to as the ICP reverse 'gas' model, it is important to note that ICP does not use 'gas' -- it uses cycles.
+It is important to note that ICP does not use 'gas' -- it uses cycles.
```

Comment:
... ICP does not use the term 'gas', it uses the term 'cycles'.

Reply:
Revised to "ICP does not use the term 'gas', it uses the term 'cycles'."